CIScripts/Tests/TCPCommunicationTest.ps1

```diff
@@ -1,3 +1,67 @@
 function Test-TCPCommunication {
-    # TODO: JW-838: Implement
+    Param ([Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session,
+           [Parameter(Mandatory = $true)] [TestConfiguration] $TestConfiguration)
+
+    Write-Host "Running TCP Communication test"
+
+    Initialize-TestConfiguration -Session $Session -TestConfiguration $TestConfiguration
+
+    $NetworkName = $TestConfiguration.DockerDriverConfiguration.NetworkConfiguration.NetworkName
+
+    Write-Host "Creating containers"
+    $ServerID, $ClientID = Invoke-Command -Session $Session -ScriptBlock {
+        $ServerID = docker run --network $Using:NetworkName -d iis-tcptest
+        $ClientID = docker run --network $Using:NetworkName -d microsoft/nanoserver ping -t localhost
+        return $ServerID, $ClientID
+    }
+
+    . $PSScriptRoot\CommonTestCode.ps1
```

Comment:
Wouldn't `Import-Module` be a better way (file-system independant) to provide the same functionality?